da-indexer/da-indexer-migration/src/m20220101_000001_create_table.rs

```diff
@@ -0,0 +1,40 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let sql = r#"
+            CREATE TABLE "celestia_blobs" (
+                "id" bytea PRIMARY KEY,
+                "height" bigint NOT NULL,
```

Comment:
Nit: just interested, why not to create a foreign key reference from `celestia_blobs.height` to `celestia_blocks.height`?

Reply:
The main reason why it is missing now is the order of writes in the db. Currently, we insert blobs before inserting the block's metadata. We can't reverse this order within the current implementation since we rely on the blocks table to determine missing data. I guess we might make these inserts atomic if you think that it is necessary.

Reply:
Fixed in [dee226b](https://github.com/blockscout/blockscout-rs/pull/831/commits/dee226bfe4f15405cb3dfb3b0354a2f436705296)